html/semantics/links/downloading-resources/header-origin-no-referrer.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Ping attribute Origin Header No Referrer Policy</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <meta name='referrer' content='no-referrer'>
```

Comment:
Wouldn't https://fetch.spec.whatwg.org/#append-a-request-origin-header cause the origin to be null here?

Reply:
I think you are right. So setting referrer to no-referrer only has an effect on the request's referrer, not its origin header, correct? I updated header-origin-no-referrer.html.

Reply:
Sorry for the confusion, I meant "setting referrer to no-referrer" in this context )not the meta tag):
https://html.spec.whatwg.org/multipage/links.html#hyperlink-auditing

Reply:
It has an effect on the request's `Origin` header as well as the request's `Referer` header. (It does not always affect the `Origin` header, that algorithm defines when it does.)

Does that help?

Reply:
You mean https://fetch.spec.whatwg.org/#append-a-request-origin-header right, in case of not GET/HEAD? Right in my examples only no-referrer case is triggered, as the tests are same-origin and same protocol.